genomesubsampler/prototypeSelection.py

```diff
@@ -180,28 +238,32 @@ def prototype_selection_constructive_maxdist(dm, num_prototypes):
     def prototype_selection_constructive_maxdist(dm: DistanceMatrix,
     num_prototypes: int) -> List[str]:
```

Comment:
I don't understand. I am speaking about adding the sting ', seedset: List[str]' to the signature in the docstring.